decidim-conferences/app/constraints/decidim/conferences/current_component.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
```

Comment:
@decidim/lot-core shouldn't this class be generalized into `decidim-core` and maybe take some configuration params or allow inheriting from it?

Reply:
Maybe... 
But it's out of scope of this PR

Reply:
Can you open an issue to perform this task, please?